modules/tests/src/test/scala-2.13+/higherkindness/droste/tests/SchemePartialBasisTests.scala

```diff
@@ -96,7 +98,7 @@ class SchemePartialBasisTests extends Properties("SchemePartialBasis") {
       if (n > 0) Mu(Some(expected(n - 1)))
       else Mu(None: Option[Mu[Option]])
 
-    forAll((n: Int Refined Less[W.`100`.T]) => f(n) ?= expected(n))
+    forAll((n: Int Refined Less[100]) => f(n) ?= expected(n))
```

Comment:
In scala3, the `W` witness syntax isn't supported, so I moved these to `src/test/scala-2.13+` and used the literal type syntax instead.

Thanks to Rob Norris (as always) for help on that;
https://github.com/tpolecat/skunk/blob/main/build.sbt#L64-L73